bpf/lib/encap.h

```diff
@@ -237,6 +240,13 @@ encap_and_redirect_lxc(struct __ctx_buff *ctx, __u32 tunnel_endpoint,
 	if (!tunnel)
 		return DROP_NO_TUNNEL_ENDPOINT;
 
+#if defined(TUNNEL_MODE) && defined(ENCRYPTION_STRICT_MODE)
+	if (!strict_allow(ctx))	{
+		return send_drop_notify_error(ctx, 0, DROP_UNENCRYPTED_TRAFFIC,
+						CTX_ACT_DROP, METRIC_EGRESS);
```

Comment:
this is not properly aligned under the first line, the arguments in the second line should be adjusted via tabs, and 
if not exactly matching replace the last tab with enough spaces, to start below the first line's arguments (i.e. start at ctx), for example check how __encap_with_nodeid() is aligned in __encap_and_redirect_lxc() above (line 205)